app/src/main/java/com/cornellappdev/android/eatery/MainListFragment.java

```diff
@@ -127,16 +124,13 @@ public void onClick(View v) {
                 handleCampusPillPress();
             }
         });
-        if (mListPresenter.getCollegeTownEateriesLoaded()) {
-            mCtownPillHolder.setOnClickListener(new View.OnClickListener() {
-                @Override
-                public void onClick(View v) {
-                    handleCollegetownPillPress();
-                }
-            });
-        } else {
-            mCtownPillHolder.setEnabled(false);
-        }
+        mCtownPillHolder.setOnClickListener(new View.OnClickListener() {
```

Comment:
i don't think we can assume that collegetown eateries is parsed correctly into cache

Reply:
I threw this collegetownEateriesLoaded variable in awhile ago and needed to remove it with this PR. What happened before was that if the collegetown eateries were not loaded, it was disabling the pill so it couldn't be clicked at all. I've noticed recently that this actually leads to really weird UI as people are trying to click the pill and end up clicking the eatery behind it as the pill inter-actability is disabled. So what happens now is that upon clicking the pill and the eateries are not loaded it just moves to the collegetown screen with a blank page. I do think that more testing should be done to ensure the functionality is as intended

Reply:
imo, displaying a blank screen for ctown eateries is a worse user experience than clicking on a cornell eatery. is it possible to hide the pill if ctown eateries are not loaded?